devicon.json

```
@@ -6383,6 +6383,19 @@
         "color": "#4878be",
         "aliases": []
     },
+    {
+        "name":"notion",
+        "tags":[
+            "project-management"
+        ],
+        "versions":{
+            "svg":[
+                "original"
+            ]
+        },
+        "color":"#ffffff",
+        "aliases":[]
```

Add altnames and add space between keys and values
```suggestion
        "name": "notion",
        "altnames": [],
        "tags": [
            "project-management"
        ],
        "versions": {
            "svg":[
                "original"
            ]
        },
        "color": "#fff",
        "aliases": []
```